uctestcase.cpp ignores qWaitForWindowExposed result

Was building `bionic` branch on Arch GNU/Linux, faced this issue:
```
plugin/uctestcase.cpp:55:37: error: ignoring return value of ‘bool QTest::qWaitForWindowExposed(QWindow*, int)’, declared with attribute nodiscard [-Werror=unused-result]
         QTest::qWaitForWindowExposed(this);
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
In file included from /usr/include/qt/QtTest/qtest.h:349,
                 from /usr/include/qt/QtTest/QtTest:11,
                 from plugin/uctestcase.h:24,
                 from plugin/uctestcase.cpp:19:
/usr/include/qt/QtTest/qtestsystem.h:134:42: note: declared here
     Q_REQUIRED_RESULT inline static bool qWaitForWindowExposed(QWindow *window, int timeout = 5000)
                                          ^~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
```

I assume that it was not declared as `nodiscard` before, that's why the build broke now.

--- a/src/imports/Test/plugin/uctestcase.cpp
+++ b/src/imports/Test/plugin/uctestcase.cpp
@@ -55,7 +55,8 @@ UbuntuTestCase::UbuntuTestCase(const QString& file, ResizeMode resize, bool asse
     }
     if (rootObject()) {
         show();
-        QTest::qWaitForWindowExposed(this);
+        bool window_shown = QTest::qWaitForWindowExposed(this);
+        Q_ASSERT(window_shown);
     }
 }
 

--- a/tests/unit/recreateview/tst_recreateview.cpp
+++ b/tests/unit/recreateview/tst_recreateview.cpp
@@ -51,7 +51,8 @@ void RecreateViewTest::testCase_recreateView()
     view->setSource(QUrl::fromLocalFile("SimpleApp.qml"));
     QVERIFY(view->rootObject() != 0);
     view->show();
-    QTest::qWaitForWindowExposed(view);
+    bool window_shown = QTest::qWaitForWindowExposed(view);
+    Q_ASSERT(window_shown);
     delete view;
 
     view = new QQuickView;
@@ -60,7 +61,8 @@ void RecreateViewTest::testCase_recreateView()
     view->setSource(QUrl::fromLocalFile("SimpleApp.qml"));
     QVERIFY(view->rootObject() != 0);
     view->show();
-    QTest::qWaitForWindowExposed(view);
+    window_shown = QTest::qWaitForWindowExposed(view);
+    Q_ASSERT(window_shown);
     delete view;
 }
 

